app/backend/src/couchers/models/users.py

```diff
@@ -399,11 +399,25 @@ class User(Base):
 
     @hybrid_property
     def has_completed_profile(self) -> bool:
-        return self.avatar_key is not None and self.about_me is not None and len(self.about_me) >= 150
+        # Check if user has at least one photo in their profile gallery
+        has_photo = self.profile_gallery is not None and len(self.profile_gallery.photos) > 0
```

Comment:
This will fetch the full profile gallery, need to replace with an `exists` or similar